rbi/sorbet/builder.rbi

```diff
@@ -21,8 +21,8 @@ class T::Private::Methods::DeclBuilder
   sig {params(claz: T.untyped).returns(T::Private::Methods::DeclBuilder)}
   def bind(claz); end
 
-  sig {params(params: T.untyped).returns(T::Private::Methods::DeclBuilder)}
-  def params(**params); end
+  sig {params(unused_positional_params: T.untyped, params: T.untyped).returns(T::Private::Methods::DeclBuilder)}
+  def params(*unused_positional_params, **params); end
```

Comment:
Out of curiosity, is there a reason why everything in this file is `T.untyped`? 


Seems like a lot of these type could be pinned down more strongly:

* `type_parameters` only takes `Symbol`s
* `params` only takes `T.any(Module, T::Types::Base`), etc.